CHANGELOG.md

```diff
@@ -74,14 +74,18 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Parser
 
-#### New features
-
-- Add support for parsing the defer attribute in import statements ([#4215](https://github.com/biomejs/biome/issues/4215)).
```

Comment:
this pr finally merged again？

Reply:
I don't remember merging a new PR about this. Maybe I am wrong.

Reply:
seems merged by other pr, i will delete here.